src/commands/init/initLogic.ts

```diff
@@ -0,0 +1,25 @@
+import inquirer from 'inquirer'
+import chalk from 'chalk'
+
+export async function getUserPreferences (): Promise<any> {
+  return await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'name',
+      message: 'What is the name of your project?'
+    },
+    {
+      type: 'input',
+      name: 'database',
+      message: 'What database would you like to use?'
+    },
+    {
+      type: 'input',
+      name: 'orm',
+      message: 'What ORM would you like to use?'
+    }]);
```

Comment:
Semicolon on line 20 but nowhere else, does the linter really work?

Reply:
Haha, thanks for noticing! The linter on my part does not seem to be linting automatically...